presto-main/src/main/java/com/facebook/presto/operator/FileFragmentResultCacheConfig.java

```diff
@@ -148,4 +150,17 @@ public FileFragmentResultCacheConfig setMaxCacheSize(DataSize maxCacheSize)
         this.maxCacheSize = maxCacheSize;
         return this;
     }
+
+    public boolean isInputDataStatsEnabled()
+    {
+        return inputDataStatsEnabled;
+    }
+
+    @Config("fragment-result-cache.input-data-stats-enabled")
```

Comment:
I wonder in which cases we would not like to have it?

Reply:
Changing the input bytes to non-zero value for 100% FRC cache hit rate..is a behavior change which might break other existing oss systems. So its better and safeguard this new feature behind a flag.